src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -49,6 +50,10 @@ export interface MyCollectionArtworkFormProps {
 export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = ({
   artwork,
 }) => {
+  const {
+    deleteCollectedArtwork,
+    saveCollectedArtwork,
```

Comment:
**suggestion:** do we maybe have some patterns around naming here, this might be confusing to reviewers or someone reading the code later who didn't see where the method is coming from. Maybe something like:
```suggestion
    deleteCollectedArtwork: trackDeleteCollectedArtwork,
    saveCollectedArtwork: trackSaveDeletedArtwork,
    saveCollectedArtwork,
```